onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -693,6 +696,8 @@ private static ContentValues toContentValues(ObaRegion region) {
                 region.getSupportsObaRealtimeApis() ? 1 : 0);
         values.put(ObaContract.Regions.SUPPORTS_SIRI_REALTIME,
                 region.getSupportsSiriRealtimeApis() ? 1 : 0);
+        values.put(ObaContract.Regions.SUPPORTS_EMBEDDED_SOCIAL,
```

Comment:
I believe Embedded Social should be moved to the last line here, as it's the most recently added database field.